webrender/src/resource_cache.rs

```diff
@@ -709,7 +709,7 @@ impl ResourceCache {
                         is_opaque: image_descriptor.is_opaque,
                     }
                 } else {
-                    image_template.descriptor.clone()
+                    image_template.descriptor
```

Comment:
@nox I think you are overly sensitive here. It doesn't appear to me that the change bears any danger, and it's a bit related to the `clone()` refactor.